Setup/UpgradeData.php

```diff
@@ -0,0 +1,140 @@
+<?php
+
+/**
+ * Update data for update
+ *
+ * If you want to add improvements, please create a fork in our GitHub:
+ * https://github.com/myparcelbe
+ *
+ * @author      Richard Perdaan <info@sendmyparcel.be>
+ * @copyright   2010-2019 MyParcel
+ * @license     http://creativecommons.org/licenses/by-nc-nd/3.0/nl/deed.en_US  CC BY-NC-ND 3.0 NL
+ * @link        https://github.com/myparcelbe/magento
+ * @since       File available since Release v3.0.0
+ */
+
+namespace MyParcelBE\Magento\Setup;
+
+use Magento\Framework\Setup\ModuleContextInterface;
+use Magento\Framework\Setup\ModuleDataSetupInterface;
+use Magento\Framework\Setup\UpgradeDataInterface;
+
+/**
+ * Upgrade Data script
+ * @codeCoverageIgnore
+ */
+class UpgradeData implements UpgradeDataInterface
+{
+    /**
+     * Upgrades data for a module
+     *
+     * @param \Magento\Framework\Setup\ModuleDataSetupInterface $setup
+     * @param \Magento\Framework\Setup\ModuleContextInterface   $context
+     */
+    public function upgrade(ModuleDataSetupInterface $setup, ModuleContextInterface $context)
+    {
+        if (version_compare($context->getVersion(), '3.0.0', '<=')) {
+            $connection = $setup->getConnection();
+            $table      = $setup->getTable('core_config_data');
+            if ($connection->isTableExists($table) == true) {
+                /*
+                 * Move shipping_methods to myparcelbe_magento_general
+                 */
+                $selectShippingMethodSettings = $connection->select()->from(
+                    $table,
+                    ['config_id', 'path', 'value']
+                )->where(
+                    '`path` = "myparcelbe_magento_checkout/general/shipping_methods"'
+                );
+
+                $ShippingMethodData = $connection->fetchAll($selectShippingMethodSettings);
```

Comment:
Wat krijg je terug van `fetchAll()` als je null of false krijgt, dan kan je het volgende doen:

`$ShippingMethodData = $connection->fetchAll($selectShippingMethodSettings) ?? [];`
Dan kan die if statement weg

Als je een lege array terug krijgt, kan nu ook al die if statement weg.

Zie ook ander code hieronder

Reply:
Done